### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by lightspeed69**

### CODE REVIEW

File: test/test_point.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 from vector import Vector
 
 
 class TestLineSegment:
     def test_vector_property(self):
-        a = Point(1, 1)
-        b = Point(2, 1)
-        vector = LineSegment(a, b).vector
-        assert vector == Vector(1, 0)
+        assert LineSegment(Point(1, 1), Point(2, 1)).vector == Vector(1, 0)
```
Comment (valentynmalyi) [inline test/test_point.py:9]: rename `test_vector_property` -> `test_vector`.
All test names must be without `property`

File: test/test_point.py [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+
+    def test_square_length_property(self):
+        assert LineSegment(Point(1, 1), Point(2, 2)).vector.square_length == 2
+
```
Comment (valentynmalyi) [inline test/test_point.py:13]: You don't test LineSegment propety. Here you test vector property

**Comment by valentynmalyi** [comment]
Also install 
flake8-test-name

File: test/test_point.py (hunk 1 above)
Comment (lightspeed69) [reply test/test_point.py:9]: done

File: test/test_point.py (hunk 2 above)
Comment (lightspeed69) [reply test/test_point.py:13]: fixed

**Pull request merged**